source/administration/changelog.md

```diff
@@ -4,11 +4,233 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+- Added "View Members" option to Town Square.
+- Moved "Start Video Call" option to the bottom of the profile popover.
+- Added a confirmation dialog when leaving a private channel.
+- User preferences such as display settings now sync between browser tabs, between different browsers, and across devices.
+
+#### Performance
+- Added the ability to isolate searches to specific read-replicas for full text search queries for higher performance.
+- Added default read and write timeouts for MySQL datasource for ... // XXX check with engineers
+- Added password field to the [bulk import tool](https://docs.mattermost.com/deployment/bulk-loading.html).
+- Added the ability to disable full text search queries and statuses via `config.json` for higher performance.
+
+#### Emoji Picker (Beta)
+- Enable the emoji picker in **Account Settings > Advanced > Preview pre-release features**. 
+- Custom emoji now maintains aspect ratio in the emoji picker.
+- Improved user experience for closing the Emoji picker after reacting to a message.
+
+#### Channel Switcher (CTRL/CMD+K)
+- Pressing ENTER once in the channel switcher (CTRL/CMD+K) now switches the channel.
+- Using a mouse to select a channel in the channel switcher (CTRL/CMD+K) now switches to the correct channel.
+
+#### Markdown Text Formatting
+- Added a margin for Markdown inline images.
+- Improved Markdown heading sizes in the desktop view.
+
+#### On-Boarding
+- Added "Already have an account? Click here to sign in" link to the sign up page.
+- Improved experience of joining a team using an invite link.
+
+#### Files
+- SVG files now render in file preview.
+
+#### CLI Tool
+- Added new CLI commands:
+    - `platform check` for validating the `config.json` file.
+    - `platform search` for searching users based on username, email or user ID.
+
+#### OAuth 2.0 Service Provider
+- OAuth 2.0 service provider now always returns the refresh token.
+- New refresh token now issued when granting a new access token.
+
+#### System Console
+ - Added a confirmation dialog when deactivating a user.
+ - Server logs are now always printed in English regardless of Default Server Language, for easier troubelshooting.
+
+#### Enterprise Edition
+ - Added new [performance monitoring metrics](https://docs.mattermost.com/deployment/metrics.html) for
+     - The total number of connections to all the search replica databases
+     - The total number of WebSocket broadcasts sent
+
+### Bug Fixes
+- Long custom emoji names no longer float out of the emoji picker.
+- Deleted custom emoji no longer stay in "recently used" section of the emoji picker.
+- The maximum length of field "Position" increased to 64 characters. The previous limit caused problems with LDAP synchronization.
+- Pinning a post in center channel no longer changes pinned posts list in the right-hand sidebar.
+- Pinning a post in center now adds the badge to search results.
+- Fixed error message text for **Edit URL** field in channel creation modal.
```

Comment:
Is "modal" an internal dev term? Use the title of the modal, and use either "window" or "dialog" instead of "modal": the *New Channel* window